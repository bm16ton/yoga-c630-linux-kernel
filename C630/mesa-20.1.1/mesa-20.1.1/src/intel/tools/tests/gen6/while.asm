while(8)        JIP: -76                                        { align16 1Q };
while(8)        JIP: -108                                       { align1 1Q };
while(16)       JIP: -108                                       { align1 1H };
(-f0.0) while(8) JIP: -48                                       { align1 1Q };
(-f0.0) while(16) JIP: -48                                      { align1 1H };
(-f0.0.x) while(8) JIP: -48                                     { align16 1Q };
