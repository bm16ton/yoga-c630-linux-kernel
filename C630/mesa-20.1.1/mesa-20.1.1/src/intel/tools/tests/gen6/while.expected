27 01 60 00 8f 10 b4 ff 04 00 6e 00 04 00 6e 00
27 00 60 00 8f 10 94 ff 00 00 8d 00 00 00 8d 00
27 00 80 00 8f 10 94 ff 00 00 8d 00 00 00 8d 00
27 00 71 00 8f 10 d0 ff 00 00 8d 00 00 00 8d 00
27 00 91 00 8f 10 d0 ff 00 00 8d 00 00 00 8d 00
27 01 72 00 8f 10 d0 ff 04 00 6e 00 04 00 6e 00
